spec/security_characteristics_more.md

```diff
@@ -0,0 +1,59 @@
+## Addendum: [[def: More on security characteristics]]
+
+The elaboration below combines well with [[ref: Security characteristics]] above.
+
+### KEL backed data: High Security
+
+KEL backed information in `did:webs` can either be found in the KEL or found anchored to the KEL.
```

Comment:
KEL is probably the wrong term here since the data in the `keri.cesr` file combines KEL and TEL (key and non-key transactions).  I think we need to get that consistent throughout the spec.